### PR TITLE
add katello 4.0 config

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: configs/katello/4.0.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.4
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.7.3
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.3
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
```
Comment (evgeni) [inline configs/katello/4.0.yaml:40]: @jjeffers this will need updates, but I wanted to get the basic config layed out so I can branch koji

Comment (jjeffers) [reply configs/katello/4.0.yaml:40]: @evgeni ok - I'll need to see what the versions of hammer, et al, are to be. Ty!

File: configs/katello/4.0.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  :katello:
+    :main: true
+    :branch: KATELLO-4.0
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.4
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.24.2
```
Comment (jjeffers) [inline configs/katello/4.0.yaml:30]: Just confirmed with @chris1984 that the new hammer version will be 1.0

Comment (jjeffers) [reply configs/katello/4.0.yaml:30]: No other changes to other katello repos, as far as I know.

Comment (evgeni) [reply configs/katello/4.0.yaml:30]: updated, thanks!

**Review by jjeffers — APPROVED** [review]
LGTM

**Pull request merged**